sources/packages/web/src/views/institution/student/applicationDetails/AssessmentAward.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Assessments"
+        subTitle="View Assessment"
+        :routeLocation="routeLocation"
+      />
+      <detail-header :headerMap="headerMap" />
+    </template>
+    <!-- todo: should inistition be allowed to accept coe from here? -->
```

Comment:
Accepting COE here is a Ministry-exclusive operation unless there is an explicit requirement for it.
Institutions must confirm COE using the COE page.

Reply:
ya, we don't have a requirement,  I just wanted to check with @JasonCTang that's why added a Todo.